engine/src/flutter/display_list/geometry/dl_path.cc

```diff
@@ -254,24 +270,64 @@ DlPath DlPath::operator+(const DlPath& other) const {
 
 SkPath DlPath::ConvertToSkiaPath(const Path& path, const DlPoint& shift) {
   SkPath sk_path;
-  sk_path.setFillType(ToSkFillType(path.GetFillType()));
+
+  DlPathReceiver receiver{
+      .path_info =
+          [&sk_path](DlPathFillType fill_type, bool is_convex) {
+            sk_path.setFillType(ToSkFillType(fill_type));
+          },
+      .move_to =
+          [&sk_path](const DlPoint& p2) { sk_path.moveTo(ToSkPoint(p2)); },
+      .line_to =
+          [&sk_path](const DlPoint& p2) { sk_path.lineTo(ToSkPoint(p2)); },
+      .quad_to =
+          [&sk_path](const DlPoint& cp, const DlPoint& p2) {
+            sk_path.quadTo(ToSkPoint(cp), ToSkPoint(p2));
+          },
+      .conic_to =
+          [&sk_path](const DlPoint& cp, const DlPoint& p2, DlScalar weight) {
+            sk_path.conicTo(ToSkPoint(cp), ToSkPoint(p2), weight);
+            return true;
+          },
+      .cubic_to =
+          [&sk_path](const DlPoint& cp1,  //
+                     const DlPoint& cp2,  //
+                     const DlPoint& p2) {
+            sk_path.cubicTo(ToSkPoint(cp1), ToSkPoint(cp2), ToSkPoint(p2));
+          },
+      .close = [&sk_path]() { sk_path.close(); },
+  };
+
+  DispatchFromImpellerPath(path, receiver);
+
+  return sk_path;
```

Comment:
I just noticed that this method never dealt with the Shift parameter. On the other hand, this is a private static method and none of the local callers use that parameter. It should be removed.

Reply:
shift parameter removed in latest commit.